src/nvim/message.c

```diff
@@ -137,6 +139,81 @@ int msg_attr(const char *s, const int attr) FUNC_ATTR_NONNULL_ARG(1)
   return msg_attr_keep((char_u *)s, attr, false);
 }
 
+int msg_echo_attr(const char *s, const int attr) FUNC_ATTR_NONNULL_ARG(1)
+{
+  return msg_echo_attr_keep(s, attr, false);
+}
+
+int
+msg_echo_attr_keep(const char *s, const int attr, int keep)
```

Comment:
You are still separating type from function name. Correct examples of formatting are new files like src/nvim/eval/typval.c, not legacy code here.

Reply:
BTW, function returns boolean.

Reply:
And it is logical to either have `const int keep` or `int attr`. I normally prefer putting `const` everywhere I can.